content/en/tracing/trace_collection/compatibility/dotnet-core.md

```diff
@@ -114,6 +114,9 @@ The .NET Tracer works on .NET Core 2.0, 2.1, 2.2, and 3.0, but these versions re
 |-----------------------------------------------|-------------------------------------------|------------------------------------------------------------------------|-----------------------------------------|
 | JIT Compiler bug on Linux/x64                 | 2.0.x,</br>2.1.0-2.1.11,</br>2.2.0-2.2.5  | Upgrade .NET Core to the latest patch version, or follow steps in the linked issue | [DataDog/dd-trace-dotnet/issues/302][6] |
 | Resource lookup bug with a non `en-US` locale | 2.0.0                                     | Upgrade .NET Core to 2.0.3 or above                                    | [dotnet/runtime/issues/23938][7]        |
+| JIT Compiler bug causing crash on shutdown    | 2.0.0-2.2.x                               | Upgrade .NET Core to 3.1.0 or above | [dotnet/runtime/pull/11885][15]   |
```

Comment:
I can't comment there, but can you add .NET 8 to the paragraph above this table, line 111?

Reply:
I added it myself. Hope you don't mind me jumping into your PR 😝

Reply:
Haha, thanks! I actually didn't touch it, because I thought you'd already fixed that in a different PR and didn't want to cause conflicts 😂